test/functional/apps/dashboard_elements/input_control_vis/input_control_options.ts

```diff
@@ -136,6 +134,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           'inputControlEditorUpdateFiltersOnChangeCheckbox'
         );
         await PageObjects.visEditor.clickGo();
+        await PageObjects.common.unsetTime();
```

Comment:
Should this go in the `after` for the larger `input control options` test suite rather than as part of the `after` for the inner `updateFiltersOnChange` test suite? Or is it even necessary?

Reply:
makes sense added `after` for the larger `input control options` test